configure.ac

```diff
@@ -678,6 +678,8 @@ if test "$with_skey" = "yes"; then
 	]])],[AC_DEFINE(SKEY_BSD_STYLE, 1, [Define to support newer BSD S/Key API])],[])
 fi
 
+AC_CHECK_FUNC(fgetpwent_r, [AC_DEFINE(HAVE_FGETPWENT_R, 1, [Define to 1 if you have the declaration of 'fgetpwent_r'])])
```

Comment:
Please change this to "Defined".  Because "Define" implies that the user should do it.

Reply:
Should be fixed now.